### ISSUE TITLE
Inconsistent environments for `tape-run` and `tape-run -p`

### ISSUE THREAD

**Issue by bsuh** [juliangruber/tape-run#61]
In my opinion the environment between `tape-run` and `tape-run -p` should be as close as possible. Yet, running with `-p` introduces stuff like source-map-support. This can be confusing as to why source maps might work with `-p` but not headless electron.

**Comment by juliangruber** [comment]
I agree, this is confusing, and ideally electron would have source map support too
